collectionsdb.spf

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you delete this one as well? We need to export the database as an SQL file rather than SPF.

I'll show you how it's done!